Installation/Schema/zrc/zrc.relevanteZaak.schema.json

```diff
@@ -13,12 +13,11 @@
       "deprecated": true
     },
     "url": {
-      "type": "string",
+      "format": "url",
+      "description": "URL-referentie naar de ZAAK.",
+      "$ref": "https://vng.opencatalogi.nl/schemas/zrc.zaak.schema.json",
       "required": true,
-      "maxLength": 1000,
-      "minLength": 1,
-      "default": "{{ selfUrl(object) }}",
```

Comment:
Weet je zeker dat het veld url op deze manier moet werken voor een RelevanteZaak? Aangezien de meeste objecten inderdaad voor het veld url 
"default": "{{ selfUrl(object) }}",
Gebruikte

Reply:
Dit is een referentie naar een (relevante) andere zaak. Dus niet de selfUrl van het object, maar echt een verwijzing naar een zaak

Reply:
<img width="543" alt="image" src="https://github.com/CommonGateway/ZGWBundle/assets/55533449/7e26dd11-273f-4230-be7c-3fe5099a4cf2">